src/Microsoft.DotNet.TestFramework/TestAssetInfo.cs

```diff
@@ -346,7 +346,7 @@ public IEnumerable<FileInfo> LoadInventory(FileInfo file)
                 file.Refresh();
                 if (!file.Exists)
                 {
-                    throw new ArgumentException("Inventory file should exist.");
```

Comment:
Why do we get flipping on this? We had empty, then ArgumentException and back to Empty.